treeherder/webapp/api/performance_data.py

```diff
@@ -46,12 +46,18 @@ def list(self, request, project):
                 parent_signature__in=parent_signatures)
 
         if not int(request.query_params.get('subtests', True)):
-            signature_data = signature_data.filter(parent_signature__isnull=True)
+            try:
+                signature_data = signature_data.filter(parent_signature__isnull=True)
+            except ValueError:
+                return "No available subtest provided", HTTP_400_BAD_REQUEST
```

Comment:
Ed said we may got ValueError if someone use "1 and 2" as subtests https://bugzilla.mozilla.org/show_bug.cgi?id=1308122#c0